routing/message/handlers/state/shadow_state_handler_test.go

```diff
@@ -0,0 +1,166 @@
+// Copyright (c) 2023 Contributors to the Eclipse Foundation
+//
+// See the NOTICE file(s) distributed with this work for additional
+// information regarding copyright ownership.
+//
+// This program and the accompanying materials are made available under the
+// terms of the Eclipse Public License 2.0 which is available at
+// https://www.eclipse.org/legal/epl-2.0, or the Apache License, Version 2.0
+// which is available at https://www.apache.org/licenses/LICENSE-2.0.
+//
+// SPDX-License-Identifier: EPL-2.0 OR Apache-2.0
+
+package state
+
+import (
+	"fmt"
+	"testing"
+
+	"github.com/ThreeDotsLabs/watermill"
+	"github.com/ThreeDotsLabs/watermill/message"
+	"github.com/eclipse-kanto/aws-connector/config"
+	"github.com/eclipse-kanto/aws-connector/routing/message/handlers"
+	"github.com/eclipse-kanto/aws-connector/routing/message/handlers/passthrough"
+	"github.com/eclipse-kanto/suite-connector/connector"
+	"github.com/stretchr/testify/assert"
+	"github.com/stretchr/testify/require"
```

Comment:
```suggestion

	"github.com/eclipse-kanto/aws-connector/config"
	"github.com/eclipse-kanto/aws-connector/routing/message/handlers"
	"github.com/eclipse-kanto/aws-connector/routing/message/handlers/passthrough"

	"github.com/ThreeDotsLabs/watermill"
	"github.com/ThreeDotsLabs/watermill/message"
	"github.com/eclipse-kanto/suite-connector/connector"
	"github.com/stretchr/testify/assert"
	"github.com/stretchr/testify/require"
```

Reply:
I really do not know why those are not getting formatted. I thought the IDE and perhaps running the gofmt should handle even the imports formatting properly. I guess I will have to manually move them a bit.

Reply:
I really prefer this straight forward way of setting up the tests. Plus I can verify single bahaviour with single test execution. If it was just to verify that some input provides some output we could go with single test actually. Let's have it like this for now.

Thanks for the typos. 👍 

I guess those are the risks of having long method names explaining complex behaviour :D

Reply:
> I really do not know why those are not getting formatted. I thought the IDE and perhaps running the gofmt should handle even the imports formatting properly. I guess I will have to manually move them a bit.

This is more of a standard approach that all go projects adopted, but never went into the tooling